.eslintrc.js

```diff
@@ -8,7 +8,15 @@ const config = {
 	rules: {
 		...( wpConfig?.rules || {} ),
 		'jsdoc/valid-types': 'off',
+		'no-console': 'off',
```

Comment:
Not sure if it's a good idea to turn this off for the whole plugin. Perhaps we can just disable it where a `console.log` is expected instead?

Reply:
Done in 912abef